pkg/controller/pkg/revision/hook_test.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/pkg/errors"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
This looks like a dupe of the above `corev1`.

Reply:
ah good catch! fixed :+1: